### PR TITLE
[MOSIP-30646] Automated ssl certificates renewal.

### PR DESCRIPTION
**Pull request by Mohanraj209**

### CODE REVIEW

File: deployment/v3/utils/ssl_renewel_util/certs_renew_cron.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# script is the schedule cron to run the certs renewal script on the expiry date.
+
+# Prompt the user for the expiry date
+read -p "Enter the expiry date (YYYY-MM-DD): " expiry_date
```
Comment (syedsalman3753) [inline deployment/v3/utils/ssl_renewel_util/certs_renew_cron.sh:5]: use the command in the same script to fetch the expiry date for the domain

File: deployment/v3/utils/ssl_renewel_util/certs_renew_cron.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+# Convert the expiry date to a cron format (minute hour day month)
+cron_time="0 6 $(date -d "$expiry_date" +'%d') $(date -d "$expiry_date" +'%m') *"
+
+# Define the command you want to run on the expiry date
+command_to_run="renew_certificates.sh"
+
+# Create a cron job
+(crontab -l; echo "$cron_time $command_to_run") | crontab -
```
Comment (syedsalman3753) [inline deployment/v3/utils/ssl_renewel_util/certs_renew_cron.sh:14]: maintain log file for cron

File: deployment/v3/utils/ssl_renewel_util/renew_ssl_certs.sh [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    # Move the directory
+    sudo mv "$src_dir" "$dest_dir"
+    echo "Directory moved to $dest_dir"
+else
+    echo "Source directory $src_dir does not exist."
+fi
+
+# Regenerate the ssl certificates
+# Prompt the user for the domain name
+read -p "Enter the domain name (e.g., sandbox.xyz.net): " domain
```
Comment (syedsalman3753) [inline deployment/v3/utils/ssl_renewel_util/renew_ssl_certs.sh:22]: if the domain env variable exist if not prompt for domain

Comment (syedsalman3753) [reply deployment/v3/utils/ssl_renewel_util/renew_ssl_certs.sh:22]: use `mosip_domain` as env variable

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

File: deployment/v3/utils/ssl_renewel_util/certs_renew_cron.sh (hunk 1 above)
Comment (Mohanraj209) [reply deployment/v3/utils/ssl_renewel_util/certs_renew_cron.sh:5]: Updated

File: deployment/v3/utils/ssl_renewel_util/renew_ssl_certs.sh (hunk 3 above)
Comment (Mohanraj209) [reply deployment/v3/utils/ssl_renewel_util/renew_ssl_certs.sh:22]: Updated

File: deployment/v3/utils/ssl_renewel_util/certs_renew_cron.sh (hunk 2 above)
Comment (Mohanraj209) [reply deployment/v3/utils/ssl_renewel_util/certs_renew_cron.sh:14]: updated

**Pull request closed without merging**